src/bot/scripts/card.js

```diff
@@ -50,1171 +52,1143 @@ const readDir = dirPath => {
+    }
+
+    if (
+      cardData.type === "character" &&
+      cardData.can_attack &&
+      cardData.character_type !== "totem"
+    ) {
+      // card that can attack
+
+      embed.description += "**Can Attack? - Yes**\n";
+
+      embed.description += `Attack Range: ${Math.round(
+        cardData.attack_range * 100
+      ) / 100}\n`;
+      embed.description += `Knockback: ${Math.round(
+        parseInt(cardData.knockback) * 100
+      ) / 100} at ${Math.round(cardData.knockback_angle * 100) / 100}°\n\n`;
+    } else {
+      // spell, totem or card that cant attack
+
+      embed.description += "**Can Attack? - No**\n\n";
+    }
+
+    if (cardData.has_aoe && cardData.type !== "spell") {
+      // aoe attacks
+
+      embed.description += "**AOE Attacks? - Yes**\n";
+
+      embed.description += `AOE damage Percentage: ${Math.round(
+        cardData.aoe_damage_percentage * 100
+      ) / 100}\n`;
+      embed.description += `AOE Knockback Percentage: ${Math.round(
+        cardData.aoe_knockback_percentage * 100
+      ) / 100}\n`;
+      embed.description += `AOE Radius: ${Math.round(
+        cardData.aoe_radius * 100
+      ) / 100}\n\n`;
+    } else {
+      // no aoe
+
+      embed.description += "**AOE Attacks? - No**\n\n";
+    }
+
+    embed.description += `Full Stats: https://sppd.feinwaru.com/${cardData.image}`;
+
+    embed.setFooter("© 2018 Copyright: Feinwaru Software ");
+    console.log(commandValues);
+    // ***ATTACK INFO***
+
+    // -can attack
+    // -attack range
+    // -time in between attacks
+    // -pre attack delay
+    // -knockback impulse
+    // -aoe attack type
+    // -aoe damage percentage
+    // -aoe radius
+    // -aoe knockback percentage
+
+    // ***POWER INFO***
+
+    // -targeting ...
+    // -power duration
+
+    // ***SPEED INFO***
+
+    // -time to max velocity
+    // -max velocity
+
+    // ***totem ONLY***
+
+    // -health loss
+
+    // ***spell ONLY***
+
+    // *** ??? ***
+
+    // -requirements ...
+    // -child unit limit
+
+    // *name
+    // CanAttack
+    // *description
+    // *Image
+    // -ManaCost
+    // -damage
+    // -health
+    // healthLoss - character_type: totem
+    // type - if !Character, character_type === undefined
+    // Targeting ... - power radius
+    // character_type
+    // AttackRange
+    // TimeToMaxVelocity
+    // MaxVelocity
+    // TimeInBetweenAttacks
+    // PowerDuration
+    // -PowerPower
+    // -rarity
+    // -theme
+    // Requirements
+    // AOEAttacktype
+    // AOEdamagePercentage
+    // AOERadius
+    // AOEKnockbackPercentage
+    // PreAttackDelay
+    // CastArea - ownside/anywhere
+    // ChildUnitLimit
+
+    await message.channel.send("", {
+      file: path.join(__dirname, "temp", `pd-${saveDate}.png`)
+    });
+    message.channel.send(embed);
+    fs.unlink(path.join(__dirname, "temp", `pd-${saveDate}.png`), error => {
+      if (error != null && error != undefined) {
```

Comment:
simplify this to one cmp against null

Reply:
![image](https://user-images.githubusercontent.com/54243490/71325715-9049f500-24e8-11ea-9968-2543a5a16d63.png)
Like this?